docs/version0.22.md

```diff
@@ -49,7 +49,7 @@ To learn more about support for OpenJ9 releases, including OpenJDK levels and pl
 
 ### New `-XX:[+|-]PortableSharedCache` option added
 
-This option enables AOT compiled code to be generated based on a chosen set of processor features that maximizes its portability across machines. It is currently supported only on x86. The feature is turned on by default in Docker containers and can be disabled with `-XX:-PortableSharedCache`. For more information, see [`-XX:[+|-]PortableSharedCache`](xxportablesharedcache.md).
+This option enables AOT compiled code to be portable across CPUs of different microarchitectures and portable across OpenJ9 Compressed Reference builds with a heap size in the range of 0 GB to 28 GB. It is currently supported only on x86. This portable AOT feature has a steady-state throughput penalty of 1% to 2% for OpenJ9 Compressed Reference builds with a heap size in the range of 0 GB to 3GB. It is turned on by default in Docker containers and can be disabled with `-XX:-PortableSharedCache`. For more information, see [`-XX:[+|-]PortableSharedCache`](xxportablesharedcache.md).
```

Comment:
This comment pertains to the 0.22 release (`docs/version0.22.md`) and it should not change since the 0.22 release was not changed.
We may want to add some words in a `docs/version023.md` file